tests/unit/utils/test_time_formats.py

```diff
@@ -0,0 +1,50 @@
+from typing import List, Optional, Union
+
+import pytest
+
+from merlin.utils import convert_timestring
+
+
+def _list_all_equal(my_list: List[str]) -> bool:
+    """Boolean check that all items in the list are the same"""
+    return all(item == my_list[0] for item in my_list)
+
+
+def check_all_equal(time_strings: List[Union[str, int]], method: Optional[str] = "HMS") -> bool:
+    """Check that time strings in a List convert to the same for the given conversion method."""
+    converted: List[str] = [convert_timestring(ts) for ts in time_strings]
+    all_equal: bool = _list_all_equal(converted)
+    return all_equal
+
+
+@pytest.mark.parametrize("time_string, expected_result, method", [
```

Comment:
this is super cool. a great improvement!

Reply:
Yeah, PyTest has some great builtin decorators and utilities!